bin/install-if-deps-outdated.js

```diff
@@ -24,7 +24,7 @@ const needsInstall = () => {
 };
 
 if ( needsInstall() ) {
-	const installResult = spawnSync( 'npm', [ 'install' ], {
+	const installResult = spawnSync( 'lerna', [ 'bootstrap', '--hoist' ], {
```

Comment:
Good catch. My gut reaction is that `lerna` is somewhat taking over part of the role of `npm` for us, and that there's some point to requiring it to be installed globablly...